src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -29,48 +27,267 @@ export const CollectorSignals: GraphQLFieldConfig<any, ResolverContext> = {
       bidCount: {
         type: GraphQLInt,
```

Comment:
random thought - it's _nice_ when numbers are typed as non-null. When there's a `null` (in addition to 0 or other valid values), that's sort of like yet another meaning (what does `null` mean here vs 0?).

For artworks not in an auction, can this still be 0? Can we add a field that explicitly represents if an artwork is in an auction (that probably already exists - `isInAuction` https://github.com/artsy/metaphysics/blob/d1419db333d2959bd8b81e362dc247013f92bf9a/src/schema/v2/artwork/index.ts#L908)?

Or maybe the signals that are auction-specific are under an `auction { bidCount ... }` grouping, within `collectorSignals` (so the entire `auction` group can be `null` in that case)?

Reply:
yeah i had the same thought about grouping auction values while slapping this together yesterday afternoon. I'll try to address these comments today.